packages/env/test/github.test.js

```diff
@@ -50,6 +50,7 @@ describe('GitHub', () => {
   describe('without an event payload', () => {
     beforeEach(() => {
       env = new PercyEnvironment({
+        PERCY_PARALLEL_TOTAL: '-1',
```

Comment:
Similar to all the other tests, let's just add common env vars such as this and `GITHUB_RUN_ID` to the top-level before each in this entire suite and fix the assertions that are already in the "has the correct properties" test. Especially since the run id will probably always be present.